src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/AzureFiles/ProtectionCheckTests.ps1

```diff
@@ -20,6 +20,7 @@ $fileShareName = "AzureFileShare;pstestfs1bca8f8e"
 $saName = "pstestsa1bca8f8e"
 $skuName="Standard_LRS"
 $policyName = "AFSBackupPolicy"
+$storageAccountId = "/subscriptions/da364f0f-307b-41c9-9d47-b7413ec45535/resourceGroups/pstestFSRG1bca8f8e/providers/Microsoft.Storage/storageAccounts/pstestsa1bca8f8e"
```

Comment:
Please don't use hard-coded storage accounts - you shoudl create any resources required in the test

Reply:
There is no support for storage dataplane SDK calls in the test framework due to which all the resources are pre created for AzureFiles workload